README.md

```diff
@@ -1 +1,5 @@
-# microverse_code_review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Great Job  @isaacpitwa you're almost there. I think it would be better to include the Authors in a ReadMe file plus a development setup for the repo.
Use this  [template](https://github.com/microverseinc/readme-template)

Reply:
- [ ] Good job 💪🏿 pointing this out to the student, your comment is polite, always add why you are suggesting changes in this you can tell the student that it would make their readme look more professional 👍🏿